docs/content/latest/benchmark/tpcc.md

```diff
@@ -0,0 +1,101 @@
+---
+title: TPCC
+linkTitle: TPCC
+description: TPCC
+image: /images/section_icons/architecture/concepts.png
+headcontent: Benchmark YugabyteDB using TPCC.
+menu:
+  latest:
+    identifier: benchmark-tpcc
+    parent: benchmark
+    weight: 740
+aliases:
+  - /benchmark/tpcc/
+showAsideToc: True
+isTocNested: True
+---
+
+
```

Comment:
Lets add a brief description of what this page includes. Something like
"This page will show you how to run TPCC benchmark on YSQL."

Reply:
Done.